src/NexusMods.App.UI/Pages/LibraryPage/LibraryViewModel.cs

```diff
@@ -311,15 +261,14 @@ await Parallel.ForAsync(
 
     private LibraryItemId[] GetSelectedIds()
     {
-        var ids1 = Adapter.SelectedModels
-            .OfType<IIsParentLibraryItemModel>()
-            .SelectMany(static model => model.LibraryItemIds);
-
-        var ids2 = Adapter.SelectedModels
-            .OfType<IIsChildLibraryItemModel>()
-            .Select(static model => model.LibraryItemId);
+        var ids = Adapter.SelectedModels
+            .Select(static model => model.GetOptional<LibraryComponents.InstallAction>(LibraryColumns.Actions.InstallComponentKey))
+            .Where(static optional => optional.HasValue)
+            .SelectMany(static optional => optional.Value.ItemIds)
+            .Distinct()
+            .ToArray();
```

Comment:
This doesn't select installed items for Removal command

Reply:
What mods do you use to repro this?

Reply:
Mods installed with a collection

Reply:
Resolved in a call, issue was a subscription not being active thus IDs collection was not populated.